### DISCUSSION TITLE
Choix arbitraire de la langue utilisée

### DISCUSSION THREAD

**Discussion by mknxdev** [AmauriC/tarteaucitron.js#786] [Ideas]
Re-bonjour,

Je me permets de faire une autre suggestion en plus de ma première : il serait vraiment bien de pouvoir définir arbitrairement la langue utilisée par l'outil tarteaucitron.

En regardant le code source, je vois que cette dernière est automatiquement définie par l'outil en fonction du langage utilisé par le client (le navigateur), ce qui est très bien. Mais une pratique courante est d'utiliser cette fonctionnalité en tant que fallback, dans le cas où une langue n'a pas été définie explicitement par le développeur qui utilise l'outil, qui dans ce cas prend généralement la priorité sur tout autre fonctionnalité automatique de détection de la langue.

Voilà, merci encore :)

**Comment by nicozerr** [comment]
Bonjour,

[La réponse se trouve ici.](https://github.com/AmauriC/tarteaucitron.js/issues/353#issuecomment-536913252)

Bien à vous,

**Reply by mknxdev** [reply]
Oui merci, j'ai trouvé cette variable en cherchant un peu.
Il serait bien cependant de l'indiquer dans la doc du README du coup.
